companies/bitcoinsuisse.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "bitcoinsuisse",
+    "relevant-countries": [
+        "de",
+        "at",
+        "ch"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Bitcoin Suisse",
+    "address": "Internationale E-Mail: dataprotection@bitcoinsuisse.com",
+    "email": "philipp.mittelberger@bwb.li",
```

Comment:
Da das so ein ausgelagerter Dienstleister ist, besteht die Möglichkeit einen Satz wie folgenden einzufügen?

> Meine Anfrage bezieht sich auf die Bitcoin Suisse AG, [adresse].

Reply:
aka:
> Meine Anfrage bezieht sich auf die Bitcoin Suisse AG, Grafenauweg 12, 6300 Zug, Switzerland.

Reply:
Aktuell generieren wir ja ein "Betrifft:" als erste Zeile in der E-Mail, das haben wir uns auch für diesen Zweck ausgedacht, das dürfte hier doch reichen, oder?

Reply:
klar, das klingt gut! :slightly_smiling_face: